components/ProfileView.vue

```diff
@@ -87,7 +87,8 @@ const tabs = computed<ProfileViewTab[]>(() => {
   return [
     {
       id: 'grid',
-      count: gridCount.value > 1 ? gridCount.value : 0,
+      count:
+        gridsForDisplay.value.length > 1 ? gridsForDisplay.value.length : 0,
```

Comment:
nit: I can see this is the same as before but then if you only have 1 grid for display you count as 0? 🤔  just curiosity

Reply:
Right, for single grid we don't show counter.